### DISCUSSION TITLE
Unknown flag: --use-restic

### DISCUSSION THREAD

**Discussion by erisco** [vmware-tanzu/velero#6862] [Community support Q&A] [answered]
As of Velero v1.12.0 it seems `--use-restic` is no longer an accepted flag.

```
# velero install --use-restic
Error: unknown flag: --use-restic
```

The docs for many previous versions mention this flag. I have been unable to find a change log indicating why it was removed or what to use instead.

I see there is some test utility for it still: https://github.com/vmware-tanzu/velero/blob/4bf87c01ea73a143908ca76bc845c5822c9b99a3/test/util/velero/install.go#L206
I see there is a design doc indicating it will be replaced with a `velero config` command, but that command does not exist: https://github.com/vmware-tanzu/velero/blob/4bf87c01ea73a143908ca76bc845c5822c9b99a3/design/cli-install-changes.md?plain=1#L262

How do we install Velero with Restic in v1.12.0?

**Comment by mateusoliveira43** [comment, marked as answer]
About how to use Restic with Velero 1.12.0, you need to pass `--use-node-agent` and `--uploader-type=restic` flags in `velero install` command.

Reference
- https://velero.io/docs/v1.12/file-system-backup/#install-velero-node-agent
- https://velero.io/docs/v1.12/file-system-backup/#path-selection
